rel/config/runtime.exs

```diff
@@ -0,0 +1,76 @@
+use Mix.Config
+
+defmodule Environment do
+  @moduledoc """
+  This modules provides various helpers to handle environment metadata
+  """
+
+  def get(key), do: System.get_env(key)
+
+  def get_boolean(key) do
+    case get(key) do
+      "true" -> true
+      "1" -> true
+      _ -> false
+    end
+  end
+
+  def get_integer(key, default \\ 0) do
```

Comment:
Other commands don’t have a default — I think we should return `nil` if the key doesn’t exist.

Reply:
Consistency! - @garno ™️